Standards/scs-0112-v1-volume-types.md

```diff
@@ -0,0 +1,70 @@
+---
+title: SCS Volume Types
+type: Standard
+status: Draft
+track: IaaS
+---
+
+
+## Introduction
+
+A volume is a virtual drive that is to be used by an instance (i. e., a virtual machine). With OpenStack,
+each volume is created per a type that determines basic features of the volume as provided by the backend,
+such as encryption, replication, or quality of service. As of the writing of this document, none of these
+features can be discovered by non-privileged users via the OpenStack API.
+
+## Motivation
+
+As an SCS user, I want to be able to create volumes with certain common features, such as encryption or
+replication, and to do so in a standardized manner as well as programmatically.
+This standard outlines a way of formally advertising these common features for a volume type to
+non-privileged users, so that the most suitable volume type can be discovered and selected easily--both by
```

Comment:
Missing white spaces?

```suggestion
non-privileged users, so that the most suitable volume type can be discovered and selected easily -- both by
```